spec/amp-html-layout.md

```diff
@@ -68,6 +68,8 @@ the layout of a `fill` element matches its parent.
 
 The element lets its children to define its size, much like a normal HTML `div`.
 
+### `flex-item`
+The element and other elements with layout type `flex-item` take their parent container's available space when parent has display : flex. 
```

Comment:
We need to rephrase "container's available space" to something like "container's remaining space".